recipes/relion/meta.yaml

```
@@ -0,0 +1,62 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "RELION" %}
+{% set version = "3.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/3dem/relion/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f10934058e0670807b7004899d7b5103c21ad77841219352c66a33f98586d42e
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - cmake
+    - make
+  host:
+    - {{ mpi }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - libtiff
```

```suggestion
    - libtiff
    - libpng
```

For those overlinking errors you see now :)